### DISCUSSION TITLE
What do you think about using the Release Drafter action to write release notes?

### DISCUSSION THREAD

**Discussion by AllanOricil** [scolladon/sfdx-git-delta#508] [General]
@scolladon 
https://github.com/release-drafter/release-drafter

**Comment by scolladon** [comment]
We are currently using [release-please](https://github.com/googleapis/release-please)
It seems to do pretty much the same thing but I'm not a big expert around those tools.

Do you have an opinion / experience / knowledge about both ? Which one do you prefer ?

**Comment by AllanOricil** [comment]
I didn't know about the existence of this one. Cool, I will try that out. 👍
